### ISSUE TITLE
gcloud components list does not show powershell

### ISSUE THREAD

**Issue by YnR** [GoogleCloudPlatform/google-cloud-powershell#162]
Typed gcloud components list to verify powershell was installed

```
PS C:\> gcloud components list

Your current Cloud SDK version is: 118.0.0
The latest available version is: 118.0.0

┌────────────────────────────────────────────────────────────────────────────────────────────┐
│                                         Components                                         │
├───────────────┬─────────────────────────────────────┬──────────────────────────┬───────────┤
│     Status    │                 Name                │            ID            │    Size   │
├───────────────┼─────────────────────────────────────┼──────────────────────────┼───────────┤
│ Not Installed │ Cloud Datastore Emulator            │ cloud-datastore-emulator │  16.1 MiB │
│ Not Installed │ Cloud Datastore Emulator (Legacy)   │ gcd-emulator             │  38.1 MiB │
│ Not Installed │ Cloud Pub/Sub Emulator              │ pubsub-emulator          │  10.8 MiB │
│ Not Installed │ gcloud Alpha Commands               │ alpha                    │   < 1 MiB │
│ Not Installed │ gcloud app Go Extensions            │ app-engine-go            │   1.5 MiB │
│ Not Installed │ gcloud app Java Extensions          │ app-engine-java          │ 131.0 MiB │
│ Not Installed │ gcloud app PHP Extensions (Windows) │ app-engine-php-windows   │  19.1 MiB │
│ Not Installed │ gcloud app Python Extensions        │ app-engine-python        │   7.2 MiB │
│ Not Installed │ kubectl                             │ kubectl                  │   8.1 MiB │
│ Installed     │ BigQuery Command Line Tool          │ bq                       │   < 1 MiB │
│ Installed     │ Cloud SDK Core Libraries            │ core                     │   4.0 MiB │
│ Installed     │ Cloud Storage Command Line Tool     │ gsutil                   │   2.6 MiB │
│ Installed     │ Default set of gcloud commands      │ gcloud                   │           │
│ Installed     │ gcloud Beta Commands                │ beta                     │   < 1 MiB │
└───────────────┴─────────────────────────────────────┴──────────────────────────┴───────────┘
To install or remove components at your current SDK version [118.0.0], run:
  $ gcloud components install COMPONENT_ID
  $ gcloud components remove COMPONENT_ID

To update your SDK installation to the latest version [118.0.0], run:
  $ gcloud components update
```

subsequent gcloud components install powershell only indicates that all components are up to date.

```
PS C:\> gcloud components install powershell

All components are up to date.
```

**Comment by chrsmith** [comment]
We should fix this, but it's blocked by some other stuff first.

This is because the "powershell" component of the Cloud SDK is listed as hidden. Until we have a better solution for how to register cmdlets after you install the component outside of the Cloud SDK, we'll probably want to keep it that way. (As manually typing in `gcloud components install powershell` doesn't leave your machine ready to use the cmdlets, you still manually need to register them.)

**Comment by chrsmith** [comment]
Closing out as `Won't Fix` for now. until we wire `components install powershell` to properly initialize the PowerShell cmdlets, we shouldn't advertise that as an option. (And instead, require users to generally go through the Windows Installer experience.)

With the open-source and cross-platform PowerShell runtime, we'll probably revisit the gcloud/PowerShell integration in a quarter or two. At that time (when we are mucking with the Cloud SDK runtime) we'll be able to do the necessary work to make `gcloud component install powershell` work correctly on Windows.
